src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/PropertyNamesShouldNotMatchGetMethods.cs

```diff
@@ -82,6 +82,13 @@ private static void AnalyzeSymbol(SymbolAnalysisContext context)
                 var exposedMembers = type.GetMembers(identifier).Where(member => ExposedAccessibilities.Contains(member.DeclaredAccessibility));
                 foreach (var member in exposedMembers)
                 {
+                    // Ignore Object.GetType, as it's commonly seen and Type is a commonly-used property name.
+                    if (member.ContainingType.SpecialType == SpecialType.System_Object &&
+                        member.Name == "GetType")
```

Comment:
:bulb: `nameof` here would be referring to the correct method

Reply:
Updated.